configure.sh

```diff
@@ -63,4 +74,42 @@ cd /home/vagrant/torque-sites/base/simplesaml/ansible
 envsubst < inv/local/group_vars/all.tmpl > inv/local/group_vars/all
 ansible-playbook simplesaml.yml -i inv/local
 
+# Set up ETL
+if [ -z "$OTS_SVN_USERNAME" ]
```

Comment:
Aha!  Okay, so we test whether or not it's the empty string (and if the variable isn't set, then it gets treated here as though it were the empty string).